src/xcode/ENA/ENA/Source/RootCoordinator.swift

```diff
@@ -277,6 +277,7 @@ class RootCoordinator: NSObject, RequiresAppDependencies, UITabBarControllerDele
 		diaryCoordinator.viewController.tabBarItem = diaryTabBarItem
 
 		tabBarController.tabBar.tintColor = .enaColor(for: .tint)
+		tabBarController.tabBar.unselectedItemTintColor = .enaColor(for: .textPrimary2)
```

Comment:
Does this revert the change done in https://github.com/corona-warn-app/cwa-app-ios/pull/3144 (translucent tab bar to match iOS native apps)?